feature_engine/outliers/base_outlier.py

```diff
@@ -65,6 +65,10 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         -------
         X_new: pandas dataframe of shape = [n_samples, n_features]
             The dataframe with the capped variables.
+
+            **Note:** for 'Winsorizer()' with 'add_indicators=True', the
+            output shape will be '[n_samples, 2*n_features]' or
+            '[n_samples, 3*n_features]' depending on the tail mode.
```

Comment:
Please remove this lines in the docstrings, we should add it instead to the Winsorizer class.